lms/djangoapps/courseware/date_summary.py

```diff
@@ -67,8 +66,12 @@ def link_text(self):
 
     @property
     def time_zone(self):
-        """The time zone to display in"""
-        return get_user_time_zone(self.user)
+        """
+        The time zone to display in -- defaulted to UTC
```

Comment:
Nit: while we're in this docstring - `The time zone in which to display`, and I prefer `defaults` to `defaulted`, but that might just be me.